### ISSUE TITLE
Agree on way to name events in Event Core (Type vocabulary)

### ISSUE THREAD

**Issue by Daphnisd** [iobis/env-data#4]
There might be an issue for situations, where essential metadata such as coordinates and dates are not repeated at each sub-event. In those cases the human readability of the file is highly reduced. If no information is included to indicate the hierarchical level of each event, interpretation of the event file will become problematic. Hence, OBIS could recommend filling out the DwC column type to specify the level for each event. Proposed values include “cruise”, “station”, “sample”, and “subsample”. An additional way to ensure the readability of the event hierarchy could be to build a hierarchical structure into the eventID itself. For example, one could name an eventID referring to a subsample “station5:sample1:subsample2”, with parenteventID “station5:sample1,” with parenteventID “station5”. This approach has the benefit that if the data is sorted by eventID it will also be sorted according to the hierarchical structure, which will make it easier to understand by a human reader. An additional benefit is that simply by reading the eventID one will know the level of the event record in the hierarchy, making it easier for a human reader to assess the event record separately. (from http://bdj.pensoft.net/articles.php?id=10989&instance_id=3385375)

Naming events may require developing a controlled vocabulary and maybe adding a new term to the event core. 

Any idea's or comments are welcome.

**Comment by pieterprovoost** [comment]
I'm in favor of using `type`, in addition to the recommendation to use human readable and colon separated identifiers. Are we in trouble for not using the `type` [vocabulary](http://rs.gbif.org/vocabulary/dcterms/type.xml)?

**Comment by wardappeltans** [comment]
I fully agree. I've updated the OBIS manual see http://iobis.org/manual/darwincore/#event

**Comment by Daphnisd** [comment]
https://github.com/iobis/env-data/issues/13
Proposed type vocabulary for actual samples:

“cruise”
“station”
“sample”
“subsample” 
"sizeFraction" (= special kind of subsample) 
"slice" (= special kind of subsample)

**Comment by peterdesmet** [comment]
FYI, the [DwC mapping for ESAS](https://github.com/inbo/esas2obis/blob/main/sql/dwc_event.sql) now makes use of the following types:

cruise
sample
subSample
